test/nft.test.ts

```diff
@@ -38,10 +44,28 @@ describe("NFT Module", async () => {
      * so it serves as a good test case.
      */
     try {
-      const nft = await nftModule.getWithOwner("1");
+      const nft = await nftModule.getWithOwner("0");
       chai.assert.equal(nft.owner, AddressZero);
     } catch (err) {
+
+      chai.assert.fail(err);
+    }
+  });
+
+  it("the metadata was not changed", async () => {
+    /**
+     * The token with id 1 has been burned and can never be recovered,
+     * so it serves as a good test case.
+     */
+    try {
+      const testBPS = Math.floor(Math.random() * 10) * 100;
+      await nftModule.setRoyaltyBps(testBPS);
+      //throw new Error("should not reach here");
+      //throw new Error(JSON.stringify(await nftModule.getMetadata()));
+      chai.assert.equal(testBPS, (await nftModule.getMetadata()).metadata.seller_fee_basis_points);
```

Comment:
We need to make sure that we're checking that the royalty was updated in *both* places that `setRoyaltyBps` is expected to update. I'll just push this change now because I don't want to hold this PR for any longer, but in the future that's how you would write a thorough test.